serverless/functions/how-to/create-a-function.mdx

```diff
@@ -25,19 +25,17 @@ This page shows you how to deploy your first Scaleway Function. A function is a
 1. Click **Functions** in the **Serverless** section of the side menu. The functions page displays.
 2. Click the functions namespace you want to use to create your function. 
 3. Click **Create function**. The function creation wizard displays.
-    <Lightbox src="scaleway-create-a-function.webp" alt="" />
 4. Complete the following steps in the wizard: 
     - Enter your **function code**. 
-      - From the first drop-down menu, choose to either use the inline code editor to edit your code directly from your browser in the box provided, or to upload a `*.zip` archive.
-      - From the second drop-down menu, choose your desired runtime, e.g. `node16`, `node18`, `python37`, `python310` or `go118`.
-      - From the third drop-down menu, set the Handler for your function
+      - Set the Handler for your function.
```

Comment:
```suggestion
      - Set the handler for your function.
```
@RoRoJ is right here, I checked the console & "handler" is only capitalized because it's the title for the input field. It's "handler" everywhere else.